integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -396,6 +399,86 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
     });
   });
 
+  describe("Errors", () => {
+    it("raises compensating writes errors", async function () {
+      const person1Id = new BSON.ObjectId("0000002a9a7969d24bea4cf5");
+      const person2Id = new BSON.ObjectId("0000002a9a7969d24bea4cf6");
+      const dogId = new BSON.ObjectId("0000002a9a7969d24bea4cf7");
```

Comment:
Why are these hardcoded?

Reply:
Just to have an order of the objects, I could have done the same with names probably :)

Reply:
You could generate them and then sort them manually. My concern is that if you're running the test multiple times against the same app/cluster, we could observe unexpected behavior due to old data.

Reply:
Understandable, just fixed it.